apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -5,8 +5,8 @@ import Radium from 'radium';
 import color from '@cdo/apps/util/color';
 import {AnimationCategories, CostumeCategories} from '../constants';
 var msg = require('@cdo/gamelab/locale') || require('@cdo/spritelab/locale');
-import animationLibrary from '../animationLibrary.json';
-import spriteCostumeLibrary from '../spriteCostumeLibrary.json';
+import animationLibrary from '../gamelab/animationLibrary.json';
+import spriteCostumeLibrary from '../spritelab/spriteCostumeLibrary.json';
```

Comment:
Are you keeping a list of next TODOs? Removing this dependency feels like a good one for the list of next steps.

Reply:
(and similar dependencies in other places)

Reply:
Tracking next steps in the doc for now. Fleshing that out and adding JIRA tickets is up next for me.
+1, refactoring this to not import the json files for both labs seems like a good thing to do.